<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Personnel_manager</class>
 <widget class="QDialog" name="Personnel_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>员工管理</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>491</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>员工查询和删除</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="all_personnel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>员工一览</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_q">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="query">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="view_T">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="commit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>提交更改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>删除员工</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancle_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>撤销更改</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>添加员工</string>
    </attribute>
    <widget class="QLineEdit" name="id">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>290</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="conuntry">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>230</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>国籍</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>110</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>学历</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>140</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>287</x>
       <y>140</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>员工状态</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hometown">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nation">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>260</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="emp_id">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>80</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>职务</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>200</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>籍贯</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="e_mail">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>287</x>
       <y>80</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>健康状况</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="teleno">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>170</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>生日</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="address">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>民族</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>260</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>工作岗位代号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>员工所在部门</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>110</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>参加工作时间</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="health">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>290</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>身份证号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>287</x>
       <y>170</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>家庭住址</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>307</x>
       <y>20</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>297</x>
       <y>230</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>E-mail</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>287</x>
       <y>50</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>婚姻状况</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>287</x>
       <y>200</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>联系电话</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>员工号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="affirm">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QComboBox" name="gender">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="state">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>140</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ON_JOB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OFF JOB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ON BUSINESS</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="Birthday">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="start_time">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="marriage_state">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Married</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UnMarried</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="XL">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Primary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Junior</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Senior</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bechelor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PostGraduate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Doctor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PostDoctor</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QComboBox" name="emp_dept">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="job_id">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="duty">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
